src/openscad.cc

```diff
@@ -1071,12 +1072,17 @@ int main(int argc, char **argv)
   if (vm.count("help")) help(argv[0], desc);
   if (vm.count("version")) version();
   if (vm.count("info")) arg_info = true;
+  if (vm.count("disable-manifold")) {
+    RenderSettings::inst()->manifoldEnabled = false;
+  }
 
   if (vm.count("preview")) {
     if (vm["preview"].as<string>() == "throwntogether") viewOptions.renderer = RenderType::THROWNTOGETHER;
   } else if (vm.count("render")) {
     // Note: "cgal" is here for backwards compatibility, can probably be removed soon
-    if (vm["render"].as<string>() == "cgal" || vm["render"].as<string>() == "force") {
+    if (vm["render"].as<string>() == "cgal" ||
+        vm["render"].as<string>() == "manifold" ||
```

Comment:
`--render=cgal` is just for backwards compatibility. `--render=force` is meant to convert rendered geometries into the format of the current geometry backend. Adding `"manifold"` as an option doesn't really make sense.

Reply:
Reverted this + added `--backend=(manifold|cgal)`